api.py

```
@@ -0,0 +1,600 @@
+import argparse
+import json
+import logging
+import os
+import sys
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)